backend/inspirehep/pidstore/providers/cnum.py

```diff
@@ -50,15 +52,30 @@ def next(cls, data):
         opening_date = data.get("opening_date")
         if not opening_date:
             return
-
         adjusted_date = datetime.strptime(opening_date, "%Y-%m-%d").strftime("%y-%m-%d")
         cnum = f"C{adjusted_date}"
-        cnums_count = PersistentIdentifier.query\
-            .filter(PersistentIdentifier.pid_value.like(f"{cnum}%"))\
-            .filter(PersistentIdentifier.pid_type == "cnum")\
-            .count()
+        all_postfixes = []
```

Comment:
Minor: they are suffixes, not postfixes

Reply:
due to https://en.wikipedia.org/wiki/Suffix suffix=postfix ?